src/components/Card/Card.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC, ReactNode } from 'react';
+import { K } from './types';
+import classes from './Card.module.scss';
+
+const Card: FC<{data: K}> = ({data}) => {
```

Comment:
this component is named `Card` in singular form, but although the name tells us that only one card seems to be created, it renders the same about of separate cards as the `data` array has elements.

so what's probably missing is that the component `Card` should have a few props like: title, description and title (as it was explained in the issue for this task. 

on top of that you'd need to render a component slightly differently placing "static" part of it first and rendering a list of the Card's items inside of it's content.

Reply:
1) done, but i`m really dont understand you. Because in very beggining, you created "Card component" issue, renamed it to "DishGroup" component. I`ve created "DishComponent" and again you have told me: "consider refactoring of the component, because it should be used elsewhere. so you name it Group or CardGroup or Card".
![image](https://github.com/BUI-studies/yidlorama/assets/113294189/79a7f8ea-98d2-43f3-800f-5d87e2ccfcd3)

2) Issue includes suggested props: title: string dishes: Dish[].
![image](https://github.com/BUI-studies/yidlorama/assets/113294189/e98bd7b6-8149-43be-bec6-053393ac87bf).
I will rewrite the component so that it is responsible for rendering a single card, and I will try to (as far as I understand it correctly) first render the container for the list and then the children themselves.